cli/state.go

```diff
@@ -5,10 +5,12 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"os"
 	"reflect"
 	"sort"
 	"strconv"
 	"strings"
+	"text/template"
```

Comment:
Should use `html/template`. It will most probably break some things.

Reply:
It escapes too many things, if you want to fight with it feel free to do that

Reply:
It was just one thing from what I can see.